.github/workflows/test.yaml

```diff
@@ -0,0 +1,23 @@
+name: test
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        required: true
+        type: string
+      date:
+        required: true
+        type: string
+      sha:
+        required: true
+        type: string
```

Comment:
```suggestion
```

It looks to me like these inputs aren't being used for anything. If that's intentional, they should be removed.

Reply:
These are used by https://github.com/rapidsai/workflows/pull/59. It's mostly just to align with what every other RAPIDS repo does to track what dates/SHAs are tested.